src/Trip.js

```diff
@@ -0,0 +1,18 @@
+// const moment = require('moment');
+//
+// class Trip {
+//   constructor(tripData) {
+//     this.id = tripData.id;
+//     this.userID = tripData.userID;
+//     this.travelers = tripData.travelers;
+//     this.date = tripData.date;
+//     this.duration = tripData.duration;
+//     this.status = tripData.status || 'pending';
+//     this.suggestedActivities = [];
+//   }
+//
+// }
+//
+// export default Trip
+//
+// //
```

Comment:
were you done with these yahoos? Or did you want to comment them back in?

Reply:
Done! I asked a mentor about needing this, and they said 'nope you shouldn't' so i wound up chucking the whole file in the bin,